tests/testdata/mybun-with-img-reference/porter.yaml

```diff
@@ -9,7 +9,6 @@ images:
     description: "Whalesay as a service"
```

Comment:
It seems like maybe this file isn't used anymore?

Reply:
It's referenced in `tests/integration/archive_test` , I believe